extensions/src/main/java/org/pentaho/platform/plugin/services/email/EmailService.java

```diff
@@ -73,23 +96,44 @@ public class EmailService implements IEmailService {
    */
   private File emailConfigFile;
 
+  private static final String REFRESH_TOKEN = "refresh_token";
+  private static final String AUTHORIZATION_CODE = "authorization_code";
+
+  private static final String CLIENT_CREDENTIALS = "client_credentials";
+  private static final String GRANT_TYPE = "grant_type";
+  private static final String SCOPE = "scope";
+  private static final String CLIENT_ID = "client_id";
+  private static final String CLIENT_SECRET = "client_secret";
+  private static final String CODE = "code";
+  private static final String XOAUTH = "XOAUTH2";
```

Comment:
Technically variable should be  **_XOAUTH2_**  to avoid any confusion at a quick glance, so to avoid confusion with auth 1.0